api/tests/unit/organisations/test_unit_organisations_tasks.py

```diff
@@ -469,6 +474,94 @@ def test_handle_api_usage_notifications_above_100(
     assert OrganisationAPIUsageNotification.objects.first() == api_usage_notification
 
 
+@pytest.mark.freeze_time("2023-01-19T09:09:47.325132+00:00")
+def test_handle_api_usage_notifications_for_free_accounts(
+    mocker: MockerFixture,
+    organisation: Organisation,
+    mailoutbox: list[EmailMultiAlternatives],
+) -> None:
+    # Given
+    assert organisation.subscription.is_free_plan
+    assert organisation.subscription.max_api_calls == MAX_API_CALLS_IN_FREE_PLAN
+
+    mock_api_usage = mocker.patch(
+        "organisations.tasks.get_current_api_usage",
+    )
+    mock_api_usage.return_value = MAX_API_CALLS_IN_FREE_PLAN + 5_000
+
+    get_client_mock = mocker.patch("organisations.tasks.get_client")
+    client_mock = MagicMock()
+    get_client_mock.return_value = client_mock
+    client_mock.get_identity_flags.return_value.is_feature_enabled.return_value = True
+
+    assert not OrganisationAPIUsageNotification.objects.filter(
+        organisation=organisation,
+    ).exists()
+
+    # When
+    handle_api_usage_notifications()
+
+    # Then
+    mock_api_usage.assert_called_once_with(organisation.id, "-30d")
+
+    assert len(mailoutbox) == 1
+    email = mailoutbox[0]
+    assert email.subject == "Flagsmith API use has reached 100%"
+    assert email.body == (
+        "Hi there,\n\nThe API usage for Test Org has breached "
+        "100% within the current subscription period. Please "
+        "upgrade your organisations account to ensure "
```

Comment:
```suggestion
        "upgrade your organisation's account to ensure "
```

Reply:
Ok no problem. I've fixed the other invocations as well in the other tests.